content/developer/tutorials/getting_started/07_basicviews.rst

```diff
@@ -218,7 +218,18 @@ services *OR* have a unit price which is *NOT* between 1000 and 2000'::
             ('unit_price', '>=', 1000),
             ('unit_price', '<', 2000)]
 
-.. exercise:: Add filter and Group By.
+.. note:: XML does not allow ``<`` and ``&`` to be used inside XML
+    elements. To avoid parsing errors, entity references should be used:
+    ``&lt;`` for ``<`` and ``&amp;`` for ``&``. Other entity references
+    (``&gt;``, ``&apos;`` & ``&quot;``) are optional.
+
+    Example:
+
+        .. code-block:: xml
+
+            <filter name="negative" domain="[('test_val', 'lt;', 0)]"/>
+
+.. exercise: Add filter and Group By.
```

Comment:
The missing colon makes the block being treated as a comment.
```suggestion
.. exercise:: Add filter and Group By.
```

Reply:
Thanks, not sure how that colon went missing, but fixed it.